tests/grpc/client/tx/client.go

```diff
@@ -25,8 +27,8 @@ type Client struct {
 
 	ClientContext *client.Context
 
-	keyringKeyring keyring.Keyring
-	keyringRecord  *keyring.Record
+	keyringKeyring []keyring.Keyring
```

Comment:
we can have many keys in a single Keyring, so we don't need a list here.

Reply:
Ok